includes/admin/class-edd-notices.php

```diff
@@ -310,6 +313,62 @@ public function show_notices() {
 		settings_errors( 'edd-notices' );
 	}
 
+	/**
+	 * Show a notice if debugging is enabled in the EDD settings.
+	 * Does not show if only the `EDD_DEBUG_MODE` constant is defined.
+	 *
+	 * @since 2.11.5
+	 * @return void
+	 */
+	private function show_debugging_notice() {
+		if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+			return;
+		}
+		if ( ! current_user_can( 'manage_shop_settings' ) ) {
+			return;
+		}
+		if ( ! edd_get_option( 'debug_mode', false ) ) {
```

Comment:
```suggestion
		if ( ! edd_is_debug_mode() ) {
```

Should this be used instead? Because what you have now isn't checking the `EDD_DEBUG_MODE` constant, but `edd_is_debug_mode()` will be.

Reply:
Never mind! Misunderstood something.